### PR TITLE
[Communication] - TNM - Renaming for Phone Numbers

### PR DESCRIPTION
**Pull request by minnieliu**

### CODE REVIEW

File: sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/implementation/PhoneNumbersImpl.java [hunk 1]
```diff
@@ -13,10 +12,11 @@
 import com.azure.communication.phonenumbers.implementation.models.PhoneNumbersReleasePhoneNumberResponse;
 import com.azure.communication.phonenumbers.implementation.models.PhoneNumbersSearchAvailablePhoneNumbersResponse;
 import com.azure.communication.phonenumbers.implementation.models.PhoneNumbersUpdateCapabilitiesResponse;
-import com.azure.communication.phonenumbers.models.AcquiredPhoneNumber;
+import com.azure.communication.phonenumbers.implementation.models.PurchasedPhoneNumbers;
 import com.azure.communication.phonenumbers.models.PhoneNumberCapabilitiesRequest;
 import com.azure.communication.phonenumbers.models.PhoneNumberOperation;
 import com.azure.communication.phonenumbers.models.PhoneNumberSearchResult;
+import com.azure.communication.phonenumbers.models.PurchasedPhoneNumber;
```
Comment (sacheu) [inline sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/implementation/PhoneNumbersImpl.java:19]: Why we have 2 versions of PurchasedPhoneNumber? one in `models` and one in `implementation.models` ?

**Review by sacheu — APPROVED** [review]
Looks Good.

File: sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/implementation/PhoneNumbersImpl.java (hunk 1 above)
Comment (minnieliu) [reply sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/implementation/PhoneNumbersImpl.java:19]: The one in `implementation.models` is PurchasedPhoneNumber**s** (plural), the one in `models` is PurchasedPhoneNumber (singular)

**Pull request merged**